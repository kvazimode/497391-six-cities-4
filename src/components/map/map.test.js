import React from 'react';
import renderer from 'react-test-renderer';
import Map from './map.jsx';
jest.mock(`./map.jsx`, () => `Map`);

const offerList = [
  {
    titlee: `uno`,
    coord: [12.34, 12.30]
  },
  {
    title: `dos`,
    coord: [89.1, 76.30]
  },
  {
    title: `tres`,
    coord: [40.40, 2.20]
  }
];

it(`Map renders correctly`, () => {
  const tree = renderer
    .create(<Map offerList={offerList} />,
      {
        createNodeMock: () => {
          document.createElement(`div`)
        }
      })
    .toJSON();
  expect(tree).toMatchSnapshot();
});
