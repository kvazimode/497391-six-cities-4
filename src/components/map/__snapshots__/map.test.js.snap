// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map renders correctly 1`] = `
<Map
  offerList={
    Array [
      Object {
        "coord": Array [
          12.34,
          12.3,
        ],
        "titlee": "uno",
      },
      Object {
        "coord": Array [
          89.1,
          76.3,
        ],
        "title": "dos",
      },
      Object {
        "coord": Array [
          40.4,
          2.2,
        ],
        "title": "tres",
      },
    ]
  }
/>
`;
